rs/boundary_node/ic_boundary/src/routes.rs

```diff
@@ -61,6 +60,9 @@ pub const PATH_READ_STATE: &str = "/api/v2/canister/:canister_id/read_state";
 pub const PATH_SUBNET_READ_STATE: &str = "/api/v2/subnet/:subnet_id/read_state";
 pub const PATH_HEALTH: &str = "/health";
 
+// Will be moved to ic-bn-lib
+pub const X_ERROR_TAG: http::HeaderName = http::HeaderName::from_static("x-error-tag");
```

Comment:
We need to later figure out the `x-error-tag` and `x-ic-error-cause` ambiguity maybe. Because now that we'll move to API BNs we'll expose all those `x-ic-...` headers to the outside world (now they're stripped by `ic-gateway`)

Reply:
Switched from error-tag to error-cause